src/mame/drivers/tetrisp2.cpp

```diff
@@ -697,6 +699,17 @@ void stepstag_state::vjdash_map(address_map &map)
 }
 
 
+void stepstag_state::ymz280b_map(address_map &map)
+{
+	// map.global_mask(0x3fffff);
+	map(0x000000, 0xffffff).ram().share("ymz_ram");
```

Comment:
When you have a member for finding the share, you should use it rather than a literal tag in the map.

Reply:
Could you help me improve this part of code? Since I'm not a professional programmer, I've some difficulty catching-up with the changes of MAME code recent years.
Will be really appreciated.

Reply:
Since you have a member for finding this memory share, you can use code like:
```
    map(0x000000, 0xffffff).ram().share(m_ymzram);
```

This way you avoid repeating the literal tag, and prevent the possibility of accidentally letting the tags get out of sync between the member initialisation and the memory map.

Reply:
Compiled passed. However, when run:
"Required shared pointer ':ymz_ram' not found"
...
"Fatal error: Missing some required objects, unable to proceed"